types.ts

```diff
@@ -791,8 +791,12 @@ export interface Format {
    */
   instructions?: string;
   /**
+   * to be deprecated
```

Comment:
I think "to be deprecated" isn't very valuable, and we should omit it, or actually deprecate the field now. If using the `matchers` property works today, then we should deprecate this now (aka it's not the preferred way but it still works) and then in a future version remove it.